### PR TITLE
Adds pagination

### PR DESCRIPTION
**Pull request by esentis**
Returns paginated results GET /api/movie

### CODE REVIEW

File: src/Horudom.Web.Models/PagedResult.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	{
+#pragma warning disable CA2227 // False alarm
+		public List<T> Results { get; set; } = new List<T>();
+#pragma warning restore CA2227 // Collection properties should be read only
+
+		public int Page { get; set; }
+
+		public int TotalPages { get; set; }
+
+		public int TotalMovies { get; set; }
```
Comment (akritikos) [inline src/Horudom.Web.Models/PagedResult.cs:15]: This is a Generic PagedResult object, should be TotalElements

File: src/Horudom.Web.Api/Controller/MovieController.cs [hunk 2]
```diff
@@ -24,12 +28,31 @@ public MovieController(HorudomContext ctx, ILogger<MovieController> logger)
 		}
 
 		[HttpGet("")]
-		public async Task<ActionResult<List<MovieDto>>> GetMovies()
+		public async Task<ActionResult<PagedResult<MovieDto>>> GetMovies([PositiveNumberValidator] int page, [ItemPerPageValidator] int itemsPerPage)
```
Comment (akritikos) [inline src/Horudom.Web.Api/Controller/MovieController.cs:31]: Chill out with the validators, not every parameter is expected to have one

**Review by akritikos — APPROVED** [review]
Minor issues, now open a hotfix branch and paginate every action that returrns a List/Array :)

File: src/Horudom.Web.Api/Controller/MovieController.cs (hunk 2 above)
Comment (esentis) [reply src/Horudom.Web.Api/Controller/MovieController.cs:31]: Point taken!

**Pull request merged**